lora/paho/sub.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // MqttBroker represents the MQTT broker.
 type MqttBroker interface {
```

Comment:
I know it's not relevant for this PR, but I want to propose: rename this interface to `Subscriber` (it's best practice in go when you have an interface with only one method). Also, I propose that you rename this package from `paho` to `mqtt`.

Reply:
ok!